setup.py

```diff
@@ -80,6 +80,7 @@ def get_version(version_file):
         "Programming Language :: Python :: 3.10",
         "Programming Language :: Python :: 3.11",
         "Programming Language :: Python :: 3.12",
+        "Programming Language :: Python :: 3.13",
```

Comment:
Since we are adding backwards compatible changes (adding support for a new version of Python) we should still do a minor version bump. The changes to the install requirements below would warrant a patch bump otherwise.